legacy/definition.go

```diff
@@ -440,6 +440,17 @@ func migrateAction(baseLanguage utils.Language, a Action, localization flows.Loc
 			quickReplies = TransformTranslations(legacyQuickReplies)
 		}
 
+		for lang, attachment := range media {
+			parts := strings.SplitN(attachment, ":", 2)
+			mediaType := parts[0]
+			mediaURL := parts[1]
```

Comment:
Should we enforce that it doesn't (or alternatively does) end in a `/` so we don't double slash below?

Reply:
The legacy engine assumes they don't start with `/` but I'll add this for good measure